admin/lib/solidus_admin/configuration.rb

```diff
@@ -59,6 +59,23 @@ class Configuration < Spree::Preferences::Configuration
     preference :importmap_paths, :array, default: [
       SolidusAdmin::Engine.root.join("config", "importmap.rb"),
     ]
+
+    # The method used to authenticate the user in the admin interface, it's expected to redirect the user to the login method
+    # in case the authentication fails.
+    preference :authentication_method, :string, default: :authenticate_solidus_backend_user!
+
+    # The method used to retrieve the current user in the admin interface.
+    preference :current_user_method, :string, default: :spree_current_user
+
+    # The path used to logout the user in the admin interface.
+    preference :logout_link_path, :string, default: :admin_logout_path
+
+    # The HTTP method used to logout the user in the admin interface.
+    preference :logout_link_method, :string, default: :delete
```

Comment:
In the same direction as the above comment, if we expect the adapter to define those methods, there's no need to define them in the initializer.